skype/skype-ps/skype/Get-CsOnlineApplicationInstance.md

```diff
@@ -0,0 +1,107 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: Get-CsOnlineApplicationInstance
+schema: 2.0.0
+author: kenwith
+ms.author: kenwith
+ms.reviewer:
+---
+
+# Get-CsOnlineApplicationInstance
+
+## SYNOPSIS
+Get application instance for the tenant from AAD.
+
+## SYNTAX
+
+```
+Get-CsOnlineApplicationInstance [[-Identity] <string>] [[-ResultSize] <int>] [-Tenant <guid>] [-Force] [-WhatIf] [-Confirm]  [<CommonParameters>]
+```
+
+## DESCRIPTION
+This cmdlet is used to get application instance for the tenant from AAD.
+
+## EXAMPLES
+
+### -------------------------- Example 1 --------------------------
+```powershell
+Get-CsOnlineApplicationInstance -Identity appinstance01@contoso.com
+```
+
+This example returns the metadata of application instance with UserPrincipalName "appinstance01@contoso.com".
+
+### -------------------------- Example 2 --------------------------
+```powershell
+Get-CsOnlineApplicationInstance -ResultSize 10
+```
+
+This example returns the first 10 application instances.
+
+### -------------------------- Example 3 --------------------------
+```powershell
+Get-CsOnlineApplicationInstance -Tenant 00000000-0000-0000-0000-000000000000
```

Comment:
No need to add the tenant parameter, as the scope is already limited to tenant.

Reply:
I think adding will make it more clear from visual pespective